app/controllers/concerns/verify_profile_concern.rb

```diff
@@ -3,13 +3,14 @@ module VerifyProfileConcern
 
   def url_for_pending_profile_reason
     return idv_verify_by_mail_enter_code_url if current_user.gpo_verification_pending_profile?
-    return idv_in_person_ready_to_verify_url if current_user.in_person_pending_profile?
+    return idv_in_person_ready_to_verify_url if ready_to_verify?
     return idv_please_call_url if current_user.fraud_review_pending?
     idv_not_verified_url if current_user.fraud_rejection?
   end
 
   def user_has_pending_profile?
-    pending_profile_policy.user_has_pending_profile?
+    puts "||| pending profile? #{pending_profile_policy.user_has_pending_profile?}"
+    pending_profile_policy.user_has_pending_profile? && !current_user.double_failed?
```

Comment:
Nit: Stuff like this line now betrays what the method name suggests we're doing.